### ISSUE TITLE
setZoom doesn't work

### ISSUE THREAD

**Issue by Suprefuner** [splinetool/react-spline#125]
Hi everyone, 
I tried to set the zoom value of my scene but it doesn't work, here is my code, please let me know if I'm doing wrong
```
export default function Hero() {
  const [isLoaded, setIsLoaded] = useState(true)

  const aboutRef = useRef()
  const workRef = useRef()
  const connectRef = useRef()
  const resumeRef = useRef()

  const handleOnLoad = (spline) => {
    const about = spline.findObjectByName("about")
    const work = spline.findObjectByName("work")
    const connect = spline.findObjectByName("connect")
    const resume = spline.findObjectByName("resume")

    aboutRef.current = about
    workRef.current = work
    connectRef.current = connect
    resumeRef.current = resume

    spline.setZoom(0.5)
  }

  return (
    <section  id="home" className={`relative h-screen heroBG`} >
      {isLoaded && (
        <motion.div
          initial={{ opacity: 0, y: -150 }}
          animate={{ opacity: 1, y: "12vh" }}
          transition={{ delay: 5 }}
          className="text-6xl font-extrabold md:text-[12rem] text-[#f84874] text-center "
        >
          <motion.h2 style={{ x: left }}>Text</motion.h2>
          <motion.h2 style={{ x: right }} >Text</motion.h2>
        </motion.div>
      )}
      <div className="absolute inset-0 z-10">
        <Spline scene={url} onLoad={onLoad} />
      </div>
    </section>
  )
}
```

**Comment by alex-streza** [comment]
I just tested this and it seems you wrote **onLoad** instead of **handleOnLoad**

Otherwise it seems to work, [CodeSandbox here](https://codesandbox.io/s/cranky-water-nf5wkk?file=/src/App.js)

**Comment by rmelchner** [comment]
I have a similar problem - its not updating the Zoom:

`
			this.spline.addEventListener('wheel', (event) => {
				// console.log(event);
				const zoomSpeed = 0.05;
				this.zoomLevel += event.deltaY * -zoomSpeed;

				// Set zoom limits
				let zoomLevel = Math.max(0.1, Math.min(2.0, this.zoomLevel));
				console.log(zoomLevel);

				this.spline.setZoom(zoomLevel);
			});`

**Comment by Anno328** [comment]
Same problem happen to me.
```
export default function App() {
  function onLoad(spline: any) {
    console.log("onLoad called");
    // Set camera zoom doens't work
    spline.setZoom(100);
  }
  return (
    <div className="w-screen h-screen">
      <Spline scene="/logo_v2.spline" onLoad={onLoad} />
    </div>
  );
}
```

**Comment by marcofugaro** [comment]
This has been fixed in `@splinetool/runtime` v1.7.3.
